packages/solana/lib/src/crypto/ed25519_hd_public_key.dart

```diff
@@ -6,19 +6,30 @@ import 'package:solana/src/encoder/byte_array.dart';
 import 'package:solana/src/encoder/encoder.dart';
 import 'package:solana/src/helpers.dart';
 
+final _magicWord = 'ProgramDerivedAddress'.codeUnits;
+const _maxBumpSeed = 255;
+const _maxSeeds = 16;
+const _maxSeedLength = 32;
```

Comment:
Let's put this block with private variables after the class.

Reply:
done